draft-ietf-rats-corim.md

```diff
@@ -1187,7 +1187,7 @@ The Conditional Endorsement Series Triple is used to assert endorsed values base
 Every `conditional-series-record` selection MUST select the same `mkey`s where
 every selected `mkey`'s corresponding set of keys (i.e., `mval`._key_) MUST be the same across each `conditional-series-record`.
 For example, if a selection matches on 3 `measurement-map` statements; `mkey` is the same for all 3 statements
```

Comment:
"a selection" and "3 measurement-map statements" seems like the wrong way of describing it, since a list of measurement-map with the same mkey is ill-formed. You can only have one measurement-map with a specific mkey in a list of measurement-map.

I think you mean there are 3 conditional-series-record, and for each record, they have 1 measurement-map in their list each, and that measurement map has the same mkey, and an mval of the form you state below. That would be fine. It would also be fine for each record to have more than 1 measurement-map, such as

```
[[mkey: 0, mval: {A: _variable-X_, B: _variable-Y_, C: _variable-Z_ }], [mkey: 1, mval: {D: _variable-W_}]]
```
I don't think "`mval` contains" is appropriate either, since the first mval also contains "A=_variable-X_" without the closure. You need to state that `mval` contains exactly those 3 key/value pairs and no more.